### PR TITLE
change black on ruff format

### PR DESCRIPTION
**Pull request by KuzenkovAG**
Ruff format work like black.
Replaced all black command to ruff format.
Code not changed.
Bumped minor version.
Changes added to log.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,13 @@ Please follow [the Keep a Changelog standard](https://keepachangelog.com/en/1.0.
 
 ## [Unreleased]
 
+## [3.1.0]
```
Comment (zmievsa) [inline CHANGELOG.md:8]: Let's use a bugfix version because this cannot affect our clients

File: pyproject.toml [hunk 2]
```diff
@@ -95,11 +94,6 @@ omit = [
     "tests/test_tutorial/*/schemas/*",
 ]
 
-[tool.black]
-color = true
-line-length = 120
-target-version = ["py310"]
```
Comment (zmievsa) [inline pyproject.toml:101]: Wouldn't this also affect blacken-docs? If so, we still need this!

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,13 @@ Please follow [the Keep a Changelog standard](https://keepachangelog.com/en/1.0.
...
+
+### Changed
+
+* remove black from poetry.lock, pyproject.toml, pre-commit, makefile
+* add ruff in upper files
```
Comment (zmievsa) [inline CHANGELOG.md:13]: Whenever you write a changelog, you should only specify changes in public interface. If there are none, then you can be extremely brief:

```suggestion
* Migrate from black to ruff-format
```

**Review by zmievsa — CHANGES_REQUESTED** [review]

**Comment by zmievsa** [comment]
I am afraid you have broken the symlink and now everything is failing. Would you kindly fix that?

**Comment by zmievsa** [comment]
I am afraid you have broken the symlink and now everything is failing. Would you kindly fix that?

**Comment by zmievsa** [comment]
I have deleted the symlink. Please, merge with master to fix it.

File: CHANGELOG.md [hunk 4]
```diff
@@ -5,6 +5,12 @@ Please follow [the Keep a Changelog standard](https://keepachangelog.com/en/1.0.
 
 ## [Unreleased]
 
+## [3.1.2]
+
```
Comment (zmievsa) [inline CHANGELOG.md:9]: Put it into Unreleased instead.

```suggestion
## [3.1.2]

```

**Review by zmievsa — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 2 above)
Comment (KuzenkovAG) [reply pyproject.toml:102]: it's not affect for pre-commit blacken-docs

**Pull request merged**